package/base-files/files/bin/config_generate

```diff
@@ -95,10 +95,10 @@ generate_network() {
 	local bridge=$2
 
 	json_select network
-		json_select "$1"
-			json_get_vars device macaddr protocol ipaddr netmask vlan
-			json_get_values ports ports
-		json_select ..
+	json_select "$1"
+	json_get_vars device macaddr protocol ipaddr netmask vlan
+	json_get_values ports ports
+	json_select ..
```

Comment:
```json_select``` moves current level to the specified object/array. I personally prefer the code with indentation before this change.